src/containers/profile/coursesTab/yourCourseCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import { Link } from 'react-router-dom';
+
+export const YourCourseCard = ({ type, img, name, link }) => {
```

Comment:
You can pass here an object with these fields, like:
`export const YourCourseCard = ({ course }) => {}`
Use it like `course.type` etc. And in your parent component you can call it like:
`<YourCourseCard course={course} />`
Also I think it is better to change name of this component